include/mbedtls/ecdsa.h

```diff
@@ -178,10 +213,13 @@ int mbedtls_ecdsa_verify( mbedtls_ecp_group *grp,
  * \param f_rng     The RNG function.
  * \param p_rng     The RNG parameter.
  *
- * \note            The \p sig buffer must be at least twice as large as the
- *                  size of the curve used, plus 9. For example, 73 Bytes if
- *                  a 256-bit curve is used. A buffer length of
- *                  #MBEDTLS_ECDSA_MAX_LEN is always safe.
+ * \note            The signature \p sig is expected to be ASN.1 SEQUENCE
```

Comment:
Minor: Above you write 
> is _in_ ASN.1 SEQUENCE format

but here the 'in' is missing.

Reply:
Done.